src/vs/workbench/contrib/terminal/browser/terminal.contribution.ts

```diff
@@ -552,20 +552,21 @@ actionRegistry.registerWorkbenchAction(new SyncActionDescriptor(FindPrevious, Fi
 	mac: { primary: KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.KEY_G, secondary: [KeyMod.Shift | KeyCode.F3, KeyCode.Enter] },
 }, KEYBINDING_CONTEXT_TERMINAL_FIND_WIDGET_FOCUSED), 'Terminal: Find previous', category);
 
-// Commands miht be affected by Web restrictons
+// Commands might be affected by Web restrictons
 if (BrowserFeatures.clipboard.writeText) {
 	actionRegistry.registerWorkbenchAction(new SyncActionDescriptor(CopyTerminalSelectionAction, CopyTerminalSelectionAction.ID, CopyTerminalSelectionAction.LABEL, {
 		primary: KeyMod.CtrlCmd | KeyCode.KEY_C,
+		win: { primary: KeyCode.Ctrl | KeyCode.KEY_C, secondary: [KeyCode.Ctrl | KeyCode.Shift | KeyCode.KEY_C] },
```

Comment:
Would this work?

```
primary: KeyCode.Ctrl | KeyCode.Shift | KeyCode.KEY_C,
win: { secondary: KeyCode.Ctrl | KeyCode.KEY_C },
mac: { secondary: KeyCode.Ctrl | KeyCode.KEY_C }
```

Reply:
Yes, I think that should work as well.

Reply:
I'm not sure that works but if it did we wouldn't want cmd+shift+c to be bound on mac.